bootstrap/pkg/kfapp/coordinator/coordinator.go

```diff
@@ -453,11 +453,11 @@ func (kfapp *coordinator) Delete(resources kftypes.ResourceEnum) error {
 
 	switch resources {
 	case kftypes.ALL:
-		if err := platform(); err != nil {
+		fallthrough
+	case kftypes.PLATFORM:
+		if err := k8s(); err != nil {
```

Comment:
nit: Add a comment explaining why we want to call K8s as part of platform deletion.

Reply:
added